setup.py

```diff
@@ -2,4 +2,4 @@
 
 
 if __name__ == "__main__":
-    setup()
+    setup(use_scm_version={"write_to": "src/pluggy/_version.py"})
```

Comment:
Don't we want to remove this same configuration from `pyproject.toml`?

Reply:
I was thinking to just leave it and let setuptools-scm use whichever one it wants. I don't think we're going to change this path so I'm not too concerned about someone changing one place and forgetting the other...

Reply:
i think there is a bug in setuptools scm about that, 2 parts are probably needed, lets assign me a follow-up issue as i cant work on it this week

Reply:
Should we merge this for now, or wait?

Reply:
I think we can merge, Ronny did mention "follow-up". 👍

Reply:
OK then :)